### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by biow0lf**

### CODE REVIEW

**Comment by biow0lf** [comment]
Hey @henriquegasques,

Did you need help with this gem?

**Comment by henriquegasques** [comment]
Hey @biow0lf, thanks, contributions are always welcome :)

There's a few other guys that want to help, and we are going to move the repo to an org instead of my personal profile, so I think we can update the gemspec after that. I'll use this PR as a base to do that, thanks!

File: ach.gemspec [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
+  s.name        = "ach"
   s.version     = ACH::VERSION.dup
-  s.summary     = 'Helper for building ACH files'
-  s.description = 'ach is a Ruby helper for building and parsing ACH files. In particular, it helps with field order and alignment, and adds padding lines to end of file.'
-  s.email       = 'jmorgan@morgancreative.net'
-  s.homepage    = 'https://github.com/jm81/ach'
-  s.authors     = ['Jared Morgan', 'Josh Puetz']
+  s.summary     = "Helper for building ACH files"
+  s.description = "ach is a Ruby helper for building and parsing ACH files. In particular, it helps with field order and alignment, and adds padding lines to end of file."
+  s.email       = "jmorgan@morgancreative.net"
```
Comment (henriquegasques) [inline ach.gemspec:10]: Please change this to my email:
```suggestion
  s.email       = "henriquegasques@gmail.com"
```

File: ach.gemspec [hunk 2]
```diff
@@ -1,22 +1,33 @@
...
+  s.homepage    = "https://github.com/henriquegasques/ach"
```
Comment (henriquegasques) [inline ach.gemspec:11]:
```suggestion
  s.homepage    = "https://github.com/rubyritas/ach"
```

**Pull request merged**